### PR TITLE
avatar added in component library.

### PR DESCRIPTION
**Pull request by sameep-s**
What's added:
1. Basic avatar -> circular edges and size small. className = "avatar"
2. Squares avatar -> squared off edges and same size as basic avatar. className="avatar-sq"
3. Medium sized avatars-> size of avatar is increased from small to medium for different screens and use cases. className="avatar-md"
4. Large sized avatars-> size of avatar is increased from medium to large for different screens and use cases. className="avatar-lg"

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: components/avatar.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+              />
+            </div>
+            <div class="code-container">
+              <pre><code class="html-plaintext">&lt; img src="../../img_source" class="avatar avatar-lg" &gt;</code></pre>
+            </div>
+          </div>
+        </div>
+
+        <!-- template -->
+        <!-- <div class="body-block">
```
Comment (Nagashree01) [inline components/avatar.html:277]: you can remove this comments if not needed

**Review by Nagashree01 — APPROVED** [review]
suggested some change

File: components/avatar.html (hunk 1 above)
Comment (sameep-s) [reply components/avatar.html:277]: Yes, i will remove it. thanks for the review

**Pull request merged**